src/User.php

```diff
@@ -35,20 +35,21 @@ public function getUsername(): string
 
     public function getAvatar(): string
     {
-        return $this->avatar;
+        return "./assets/imgs/avatars/" . $this->avatar;
     }
 
-    public function getBio(): string
+    public function getSanitizedBio(): string
     {
-        return $this->bio;
+        return strip_tags($this->bio);
     }
 
     public function createUserCardHtml(): string
     {
-        $htmlOutput = '<div class="userCard">';
-        $htmlOutput .= '<img class="avatarImg" src="' . $this->avatar . '" alt="Profile Picture">';
+        $htmlOutput = '<a class="profileLink" href="./userProfile.php?username=' . $this->getUsername() . '">';
+        $htmlOutput .= '<div class="userCard">';
+        $htmlOutput .= '<img class="avatarImg" src="./assets/imgs/avatars/' . $this->avatar . '" alt="Profile Picture">';
```

Comment:
because you have a method that gets the full path of the avatar, i think it would be better to use that here, that way in the future if the folder structure changes you only have to change it in one place

Reply:
Agreed, this change has been made.